company-profiles/heetch.md

```diff
@@ -0,0 +1,33 @@
+# Heetch
+
+## Company blurb
+
+Heetch is mobility solution operating across Europe and Africa. It aims to provide a profitable and comfortable experience for
+both drivers and passengers.
+
+The engineering division of the company is fully distributed and we work remotely since day 1. We're scattered all around Europe and a few other countries with a similar timezone.
+
+## Company size
+
+280
+
+## Remote status
+
+Heetch is 100% remote in the engineering division. Product division is remote as well, but people must be in a city where
+we're operating, so they can meet our users.
+
+## Region
+
+Heetch employs all across Europe and countries with similar timezones, +/- 1h.
```

Comment:
`+/- 1h` of where? 

Do you mean `+/- 1h of CET` ( Central European Standard Time )?